shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -189,6 +190,7 @@ public boolean isFailure() {
         final String segment;
         final long startOffset;
         final String serverStackTrace;
+        // This represents the offset at which the segment is truncated at.
```

Comment:
Comments like this, and on 268, and 156 should be made with the java doc style so they are applied to the getters.

Reply:
ok, the java doc style here will be applied to the getters generated by lombok. 
Updated the comments to use java doc style.